woocommerce/woocommerce-ios
# Order Detail: Disable readonly mode when opening product from order item

Handles 231-gh-woocommerce/woomobile-private


## Description

It was requested that the product form is opened when tapping on order items on the order detail screen. Currently the product form is forced to be read-only - this PR toggles this setting to make it possible for users to edit product details from the order details screen.

## Testing instructions

- Log in to the app.
- Switch to the Orders tab.
- Open an existing order or create a new one.
- On the order details screen, tap a product item.
- Notice that a product form is presented and it's editable.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/7c016848-f685-4c30-98c7-7ce55a29327d



---
